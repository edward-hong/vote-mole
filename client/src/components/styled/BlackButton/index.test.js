import React from 'react'
import { render } from '@testing-library/react'
import BlackButton from '.'

it('renders correctly', () => {
	const {
		container: { firstChild: element },
	} = render(<BlackButton>Black Button</BlackButton>)

	expect(element).toMatchSnapshot()
})
