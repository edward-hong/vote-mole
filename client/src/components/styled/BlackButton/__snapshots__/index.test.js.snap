// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  color: #21252a;
  border-color: #21252a;
}

.c0:hover,
.c0:active {
  background-color: #21252a;
  border-color: #21252a;
}

.c0:focus {
  box-shadow: 0 0 0 3px rgba(33,37,41,0.5);
}

<button
  class="c0 btn btn-secondary"
>
  Black Button
</button>
`;
